UiPath.FreeRdpClient/UiPath.SessionTools.Tests/ProgressMonitor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Nito.AsyncEx;
+
+namespace UiPath.SessionTools.Tests;
+
+internal sealed class ProgressMonitor : IProgress<string>
```

Comment:
nit: maybe a different name: eg StdOutLinesMonitor

Reply:
I called it StdMonitor.